package.json

```diff
@@ -80,5 +80,8 @@
         }
       ]
     ]
+  },
+  "dependencies": {
+    "@changesets/cli": "^2.26.2"
```

Comment:
This can be moved to `devDependencies` seeing that it is only used on local dev and CI and not when the package is installed somewhere else

Reply:
Moved this now 🙏